package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "react-native-with-moxy",
+  "version": "0.0.1",
+  "description": "The https://{project-domain} web app",
+  "author": "MOXY <hello@moxy.studio>",
+  "private": true,
+  "license": "UNLICENSED",
+  "keywords": [
+    "app",
+    "native"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:{project-org}/{project-name}"
+  },
+  "scripts": {
+    "test": "jest --coverage",
+    "start:ios:dev": "cross-env ENVFILE=.env.dev react-native run-ios --configuration Debug",
+    "start:ios:prod": "cross-env ENVFILE=.env.prod react-native run-ios --configuration Release",
+    "start:android:dev": "cross-env ENVFILE=.env.dev react-native run-android --variant devDebug",
+    "start:android:prod": "npm run build:android:js:prod && cross-env ENVFILE=.env.prod react-native run-android --variant prodRelease",
+    "build:android:js:dev": "react-native bundle --platform android --dev true --entry-file index.js --bundle-output android/app/src/main/assets/index.android.bundle",
+    "build:android:js:prod": "react-native bundle --platform android --dev false --entry-file index.js --bundle-output android/app/src/main/assets/index.android.bundle",
+    "build:android:aab:prod": "npm run build:android:js:prod && cd android && ./gradlew bundleProdRelease || cd -",
+    "build:android:aab:qa": "npm run build:android:js:prod && cd android && ./gradlew bundleQaRelease || cd -",
+    "build:android:apk": "cd android && ./gradlew assembleProdRelease || cd -",
+    "list:ios:simulators": "xcrun simctl list",
+    "list:android:simulators": "avdmanager list avd 2>/dev/null || emulator -list-avds 2>/dev/null || echo 'Not found in PATH: avdmanager and emulator'",
+    "release": "standard-version",
+    "postrelease": "git push --follow-tags origin HEAD",
+    "lint": "eslint --ignore-path .gitignore .",
+    "pods": "cd ios/ && pod install || cd -"
+  },
+  "dependencies": {
+    "@react-native-community/masked-view": "^0.1.7",
+    "@react-navigation/native": "^5.0.9",
+    "@react-navigation/stack": "^5.1.1",
+    "react": "16.9.0",
+    "react-intl": "^3.6.0",
```

Comment:
v4 is already out so we can probably start off with the newest version already 💪

Reply:
👍 at the time there was not v4 😂

Reply:
Also, to be noted that I've just added the dependency, we do not have any configuration yet as this will be done in the intl task

Reply:
Alright! I've just added a note in the task so whoever implements it will already already aware of this.